test/Microsoft.SqlTools.ServiceLayer.Test/LanguageServer/LanguageServiceTests.cs

```diff
@@ -290,6 +280,44 @@ public void AutoCompleteFindCompletions()
             Assert.True(completions.Length > 0);
         }
 
+        /// <summary>
+        /// Verify that GetSignatureHelp returns not null when the provided TextDocumentPosition
+        /// has an associated ScriptParseInfo and the provided query has a function that should
+        /// provide signature help.
+        /// </summary>
+        [Fact]
+        public async void GetSignatureHelpReturnsNotNullIfParseInfoInitialized()
+        {
+            // When we make a connection to a live database
+            ScriptFile scriptFile;
+            ConnectionInfo connInfo = TestObjects.InitLiveConnectionInfo(out scriptFile);
+
+            // And we place the cursor after a function that should prompt for signature help
+            string queryWithFunction = "EXEC sys.fn_isrolemember ";
+            scriptFile.Contents = queryWithFunction;
+            TextDocumentPosition textDocument = new TextDocumentPosition
+            {
+                TextDocument = new TextDocumentIdentifier
+                {
+                    Uri = scriptFile.ClientFilePath
+                },
+                Position = new Position
+                {
+                    Line = 1,
+                    Character = queryWithFunction.Length
+                }
+            };
+
+            // If we have a valid ScriptParseInfo and the SQL has already been parsed
+            var service = LanguageService.Instance;
+            ScriptParseInfo parseInfo = service.GetScriptParseInfo(scriptFile.ClientFilePath);
+            await service.UpdateLanguageServiceOnConnection(connInfo);
+
+            // We should get back a non-null SignatureHelp 
+            SignatureHelp signatureHelp = service.GetSignatureHelp(textDocument, scriptFile);
+            Assert.NotNull(signatureHelp);
```

Comment:
Are there any useful properties on the signatureHelp object we could also validate to make the test more complete?